Core/Core/Network/AuthEndpoint.swift

```diff
@@ -21,6 +22,8 @@ enum AuthEndpoint: EndPointType {
         switch self {
         case .getAccessToken:
             return "/oauth2/access_token"
+        case let .socialLogin(_, backend, _, _):
```

Comment:
I'm not sure but can be given the name `exchangeToken`, like the name `getAccessToken` given to login/signin.